### ISSUE TITLE
Jetbrains (inc PHPStorm)

### ISSUE THREAD

**Issue by mihaicris** [daylerees/colour-schemes#140]
Is there any chance to improve the percentage of compatibilty for Jetbrains (inc PHPStorm) which is now at 60%? Thanks.

**Comment by milesj** [comment]
I have both Intellij and PHPStorm. I could generate a large amount of the required patterns. If you could point me to any information on what interpolations are required, I can get this going.

**Comment by rjmccluskey** [comment]
👍 I'd really like to see better compatibilty here as well. I was so happy to find this repo until I tried using the themes with PhpStorm :( The colors are almost completely different than what is shown in the previews

**Comment by jcrawford** [comment]
I agree while I still use Earthsong it's not as accurate as the demo is, I was disappointed in that :(
